src/Auth0.ManagementApi/Models/SigningKey.cs

```diff
@@ -1,30 +1,85 @@
-﻿using Newtonsoft.Json;
+﻿using System;
+
+using Newtonsoft.Json;
 
 namespace Auth0.ManagementApi.Models
 {
 
     /// <summary>
-    /// 
+    /// An Application Signing Key
```

Comment:
I believe we can't just make changes to this class, as it is being used as part of the Client (https://github.com/auth0/auth0.net/blob/master/src/Auth0.ManagementApi/Models/Client.cs#L21) when calling https://auth0.com/docs/api/management/v2/#!/Clients/get_clients .

Reply:
Ah yes I absolutely missed that. Any ideas for what a new class should be called?